config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -38,6 +38,8 @@ periodics:
       env:
       - name: BOSKOS_HOST
         value: "boskos.test-pods.svc.cluster.local"
+      - name: AWS_REGION
+        value: "us-west-2"
```

Comment:
no, what is your preference for us-east-1 to us-west-2?

Reply:
I believe other AWS tests (such as kops) assume us-east-1 by default.

Reply:
concern is currently around Boskos and the janitor.

Reply:
@detiber I've checked the aws-janitor code and while it might at first look like it only reads us-east-1, it in fact will look at all resources in all available regions.

This is the kicker: https://github.com/kubernetes/test-infra/blob/master/maintenance/aws-janitor/resources/clean.go#L27-L47 and so this will be fine on us-west-2